docs/snippets/ov_auto.py

```diff
@@ -167,7 +166,7 @@ def part5():
     cpu_config = {
         hints.performance_mode: hints.PerformanceMode.LATENCY,
         streams.num: 8,
-        enable_profiling: True
+        properties.enable_profiling: True
```

Comment:
```suggestion
        enable_profiling: True
```

Reply:
we are fixing this issue in another PR https://github.com/openvinotoolkit/openvino/pull/20531
please ignore above comments.